README.md

```diff
@@ -174,6 +186,55 @@ Examples:
 
 __Note:__ Do not set `--tags` CLI parameter when running tests through the programming interface as it is internally used to pass the selected tags to the gherkin compiler.
 
+### Cucumber Expressions
+
+Besides using Regular Expressions, you can also use [Cucumber Expressions](https://cucumber.io/docs/cucumber/cucumber-expressions/) in your steps, and have support for [Optional text](https://cucumber.io/docs/cucumber/cucumber-expressions/#optional-text), [Alternative text](https://cucumber.io/docs/cucumber/cucumber-expressions/#alternative-text) and getting parameters in their desired types. The [Cucumber built-in parameter types](https://cucumber.io/docs/cucumber/cucumber-expressions/#parameter-types) are supported by default.
+
+It's also possible to add custom parameter types by creating a file that exports a `cucumberExpressions.ParameterTypeRegistry`, and passing this file's path to the CLI with `--param-type-registry-file` or to the Programming interface with the `parameterTypeRegistryFile` method.
+
+Example:
+
+1. Create a _ParameterTypeRegistry_ (e.g. _myCustomParamRegistry.js_):
+
+        const { ParameterTypeRegistry, ParameterType } = require('cucumber-expressions');
```

Comment:
Please add language hints to the code blocks:

```suggestion
    ```js
    const { ParameterTypeRegistry, ParameterType } = require('cucumber-expressions');
```

Please also do this for the other code blocks in the readme file.